src/bioregistry/data/bioregistry.json

```diff
@@ -24745,6 +24748,28 @@
     ],
     "uri_format": "http://purl.bioontology.org/ontology/CSP/$1"
   },
+  "csso": {
```

Comment:
Is this a typo? I think this should be `ccso`

Reply:
Yes, thanks for the catch. Fixed in https://github.com/biopragmatics/bioregistry/pull/1660